src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -277,7 +293,11 @@ private void launchChrome() {
      */
     private void launchFirefox() {
```

Comment:
these methods are all very similar, could they be made generic with BrowserType as parameter?